README.md

```diff
@@ -61,8 +61,15 @@ Example `config` command:
 # Adapt the path to your kubeconfig file for the garden cluster
 export KUBECONFIG=~/relative/path/to/kubeconfig.yaml
 
-# Fetch cluster-identity of garden cluster
+# Method 1 : Fetch cluster-identity of garden cluster from the configmap
 CLUSTER_IDENTITY=$(kubectl -n kube-system get configmap cluster-identity -ojsonpath={.data.cluster-identity})
+# OR
+# Method 2 : If you don't have access to the kube-system namespace in the garden cluster, the garden cluster-identity can also be extracted from every shoot's yaml
+PROJECT="your-project-name" # Change to your project name
+SHOOT="your-shoot-name" # Change to any shoot's name in your project
+PREFIX="shoot--$PROJECT--$SHOOT-"$(kubectl get shoot -n garden-$PROJECT $SHOOT -ojsonpath={.metadata.uid})"-"
+STATUS=$(kubectl get shoot -n garden-$PROJECT $SHOOT -ojsonpath={.status.clusterIdentity})
+CLUSTER_IDENTITY=$(echo ${STATUS#$PREFIX}) # difference between both
```

Comment:
```suggestion
# This function should be added to your shell profile
function getClusterIdentity () {
  local ns=$(kubectl get project $1 -ojsonpath={.spec.namespace});
  local uid=$(kubectl get shoot -n $ns $2 -ojsonpath={.metadata.uid}); 
  local status=$(kubectl get shoot -n $ns $2 -ojsonpath={.status.clusterIdentity});
  local prefix="shoot--$1--$2-$uid-"; 
  echo ${status#"$prefix"}; 
}
CLUSTER_IDENTITY=$(getClusterIdentity "your-project-name" "your-shoot-name")
```

Reply:
Why adding a function in the shell profile? This will only be needed once as it's just to configure gardenctl...

After that, it's going to be saved in `~/.garden/gardenctl-v2.yaml`

Reply:
I also don't think that we need the function in the shell profile, this should only be a simple example on how to configure / setup gardenctl.

If we really want to extract it to a function it should handle both cases (as you mentioned out of this thread) and first try to fetch it from the kube-system namespace and if it fails ask the user to input a project and shoot name to try the fallback. Only then the project name and shoot name is relevant.

But I'm not sure if it's worth the effort / makes it more complicated

Reply:
Ok. But we should not have something in our documentation where the namespace is constructed by prepending `garden-` to the project name. This must be changed.

Reply:
That's indeed a fair point! Just added an updated commit